mockgen/mockgen.go

```diff
@@ -97,9 +97,13 @@ func main() {
 	case *source != "": // source mode
 		pkg, err = sourceMode(*source)
 	case *archive != "": // archive mode
-		checkArgs()
```

Comment:
nit: now that we have `checkArgs` and `checkArgsArchive`, it becomes confusing what `checkArgs` mean. Can you rename the original `checkArgs` to `checkArgsPackage` to be consistent with `checkArgsArchive`?